source/_integrations/hifiberry.md

```diff
@@ -0,0 +1,39 @@
+---
+title: HifiBerry
+description: How to set up the HifiBerry media player platform
+logo: hifiberry.png
+ha_category:
+  - Media Player
+ha_release: 0.107.7
+ha_domain: hifiberry
+---
+
+The `HifiBerry` platform allows you to control a [HifiBerry OS](https://www.hifiberry.com/hifiberryos/) media player from Home Assistant. This is an end-to-end streaming lightweight OS built by HifiBerry for their Amp+, DAC+ or Digi+ HAT Raspberry Pi boards compatible with Airplay, Bluetooth, DLNA, LMS/Squeezebox, MPD, Snapcast, Spotify and Roon music services.
+
+To setup, you need specific configuration variables, you can add the following to your `configuration.yaml` file:
+
+```yaml
+# Example configuration.yaml entry
+media_player:
+  - platform: hifiberry
+    host: hifiberry.local
+    port: 81
```

Comment:
We try to keep all configuration samples minimal. Thus, no optional requirement in the default sample. This helps a user to get started quickly by copy-&-paste the sample without worrying about optional parameters which they most likely not need. If required, insert a full configuration sample later that covers special setups or alike.